test/ErrorHandler-test.js

```diff
@@ -1,14 +1,16 @@
 import { describe, it } from 'mocha'
+import '../src/Promise'
```

Comment:
So this was why. Indeed circular dependencies are error-prone.
Hm, does `silenceError` actually need to use an `Action` and call `_runAction` to silence the error? It seems it's only called on `Rejected` instances directly anyway.